applications/RANSApplication/python_scripts/formulations/rans_formulation.py

```diff
@@ -193,175 +171,139 @@ def IsConverged(self):
+    def GetComputingModelPart(self) -> Kratos.ModelPart:
+        """ @brief Returns computing model part used in the formulation.
+            @return Computing model part used in the formulation.
         """
         if (self.__base_computing_model_part.HasSubModelPart("fluid_computational_model_part")):
             return self.__base_computing_model_part.GetSubModelPart("fluid_computational_model_part")
         else:
             raise Exception("Computing model part \"fluid_computational_model_part\" not found as a submodel part in " + self.__base_computing_model_part.Name() + ".")
 
-    def SetMaxCouplingIterations(self, max_iterations):
-        """Sets max coupling iterations
-
-        This is not done recursively because, there are some formulations which doesn't have coupling iterations.
-        Each formulation needs to set this seperately if base class SolveCouplingStep is used.
-
-        Args:
-            max_iterations (int): Maximum number of coupling iterations to be done in the child formulations
+    def SetMaxCouplingIterations(self, max_iterations: int) -> None:
+        """ @brief Sets max coupling iterations.
+            @details This is not done recursively because, there are some formulations which doesn't have coupling iterations.
+                     Each formulation needs to set this seperately if base class SolveCouplingStep is used.
+            @param max_iterations: Maximum number of coupling iterations to be done in the child formulations.
         """
         self.__max_coupling_iterations = max_iterations
 
-    def GetMaxCouplingIterations(self):
-        """Returns maxmum number of coupling iterations used in this formulation
-
-        Returns:
-            int: Maximum number of coupling iterations
+    def GetMaxCouplingIterations(self) -> int:
+        """ @brief Returns maxmum number of coupling iterations used in this formulation.
+            @return Maximum number of coupling iterations.
         """
         if (hasattr(self, "_RansFormulation__max_coupling_iterations")):
             return self.__max_coupling_iterations
         else:
             raise Exception(self.__class__.__name__ + " needs to use \"SetMaxCouplingIterations\" first before calling \"GetMaxCouplingIterations\".")
 
-    def SetConstants(self, settings):
-        """Recursively sets constants in the formulations
-
-        Args:
-            settings (Kratos.Parameters): Constants settings
+    def SetConstants(self, settings: Kratos.Parameters) -> None:
+        """ @brief Recursively sets constants in the formulations.
+            @param settings: Constants settings.
         """
         self.__ExecuteRansFormulationMethods("SetConstants", [settings])
 
-    def GetRansFormulationsList(self):
-        """Returns list of formulations in this formulation
-
-        Returns:
-            List(RansFormulation): List of formulations in this formulation
+    def GetRansFormulationsList(self) -> "list["RansFormulation"]":
```

Comment:
```suggestion
    def GetRansFormulationsList(self) -> list[RansFormulation]:
```

Reply:
I think this is failing in python3.6... :/

Reply:
it would fail on python 3.8 as well (which is the version we currently support I think)
I just messed up and forgot about the inner `""`; I should probably stop making commits on Sunday nights :(

Reply:
> I think this is failing in python3.6... :/

that is no longer supported :tada: 

> it would fail on python 3.8 as well (which is the version we currently support I think)

yes you are right this is introduced with 3.9
you can either do `from typing import List`, or what you are doing